core/miner/main/main.cpp

```diff
@@ -441,14 +446,19 @@ namespace fc {
                                     storage_provider,
                                     retrieval_provider);
 
-    OUTCOME_TRY(api_secret, loadApiSecret(config.join("jwt_secret")));
     api::fillAuthApi(mapi, api_secret, log());
 
     std::map<std::string, std::shared_ptr<api::Rpc>> mrpc;
-    mrpc.emplace("/rpc/v0", api::makeRpc(*mapi));
+    mrpc.emplace(
+        "/rpc/v0",
+        api::makeRpc(*mapi,
+                     std::bind(mapi->AuthVerify, std::placeholders::_1)));
```

Comment:
maybe should split `verifyAuthToken(secret, token)` like `generateAuthToken` and use lambda